modules/vba_documents/lib/vba_documents/multipart_parser.rb

```diff
@@ -43,6 +42,13 @@ def self.parse_file(infile)
     end
     # rubocop:enable Metrics/MethodLength
 
+    def consume_first_line(lines)
+      lines.next[0].chomp(LINE_BREAK)
+    rescue StopIteration
+      Rails.logger.error('Tempfile was empty')
+      raise
```

Comment:
Since the `VBADocuments::UploadProcessor` doesn't specifically rescue `StopIteration` exceptions, the record will stay in "uploaded" status and be retried.